File: leetcode/10-99/26. Remove Duplicates from Sorted Array/solution_2.cs
public class Solution
{
    public int RemoveDuplicates(int[] nums)
    {
        HashSet<int> numSet = new HashSet<int>(nums);
        List<int> newNums = new List<int>(numSet);
        Array.Copy(newNums.ToArray(), nums, newNums.Count);
        return newNums.Count;
    }
}

File: leetcode/10-99/26. Remove Duplicates from Sorted Array/solution_2.java
class Solution
{
    public int removeDuplicates(int[] nums)
    {
        Set<Integer> numSet = new LinkedHashSet<>();

        for (int num : nums)
            numSet.add(num);

        int i = 0;
        for (int num : numSet)
            nums[i++] = num;

        return numSet.size();
    }
}